packages/pipeline-editor/src/CustomFormControls/validators/enum-validators.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018-2022 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Validator } from ".";
+
+export interface EnumValidatorOptions {
+  required?: boolean;
+}
+
+export function getEnumValidators<T extends string>({
+  required,
+}: EnumValidatorOptions) {
+  const validators: Validator<T>[] = [
+    {
+      enabled: required === true,
+      isValid: (value: T) => {
+        return value !== undefined && value !== "";
```

Comment:
given this represents an enum I think we should only check for undefined here, maybe empty string is a value in the enums list?